contentcuration/contentcuration/views/admin.py

```diff
@@ -141,6 +142,149 @@ def get_all_users(request):
     return Response(user_serializer.data)
 
 
+class ChannelUserListPagination(PageNumberPagination):
+    page_size = 500
```

Comment:
Might be nice to have a `PAGE_SIZE=500` constant at the top of the file for easy updating in the future